spec/lib/js_locale_helper_spec.rb

```diff
@@ -144,6 +144,10 @@ def clear_cache!
         expect(content).to_not eq("")
       end
     end
+
+    it "generates valid MF locales for the '#{locale[:value]}' locale" do
```

Comment:
This was previously done as part of the example from line 133, `JsLocaleHelper.output_locale` was generating MF locales too.